CourseApp/ClassTaskFolder/ClassTask.cs

```diff
@@ -0,0 +1,20 @@
+namespace CourseApp.Class
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class ClassTask
+    {
+        public void GetCreatedFile(string name, string extension, double weight, string weightModificator)
```

Comment:
every time you write Get - another developer expects to get something - so, please specify return value